log/logging.go

```diff
@@ -135,19 +134,14 @@ func SetPrefix(prefix string) {
 // Fatal outputs a severe error message just before terminating the process.
 // Use judiciously.
 func Fatal(id, description string, keysAndValues ...interface{}) {
-	if Level < LevelFatal {
-		return
-	}
-	DefaultLogger.logMessage(LevelFatalName, id, description, keysAndValues...)
-	osExit(1)
+	keysAndValues = append([]interface{}{"id", id}, keysAndValues...)
```

Comment:
Ok, did this, but used `golog_id=<id>` cuz of tokenized string searching issues in elasticsearch (`golog:id` would be two tokens: `golog` and `id`, but `golog_id` is a single token).